tests/sentry/grouping/snapshots/test_variants/test_event_hash_variant/newstyle@2019_04_05/malloc_sentinel.pysnap

```diff
@@ -1,5 +1,5 @@
 ---
-created: '2021-07-08T15:46:33.498715Z'
+created: '2021-07-14T14:24:10.473242Z'
```

Comment:
can you git-reset those snapshots if they only changed in datetime?

Reply:
Done.